### PR TITLE
Rename the tracks_view document to track for consistency

### PR DESCRIPTION
**Pull request by rock3r**
## Problem

The tracks view document name, `tracks_view`, is inconsistent with the other view document names. We also have some bugs in `strings.ts`.

## Solution

Rename it to `tracks`, add tests to and fix `strings.ts`.

### Test(s) added 

Yes for `strings.ts`.

### Paired with 

Nobody

### CODE REVIEW

**Comment by rock3r** [comment]
For some reason the execution always gives 304:

image

After fixing the issues in ce9c86a

File: functions/src/strings.ts [hunk 1]
```diff
@@ -1,9 +1,9 @@
 export const ensureNotEmpty = (text: string, name: string) => {
-    if (text.trim.length === 0) {
+    if (text.replace(/\s/g, '').length < 1) {
```
Comment (fourlastor) [inline functions/src/strings.ts]: The problem here is that trim is a method and you didn't invoke it, I'd go back to that and add ().

Comment (rock3r) [reply functions/src/strings.ts]: Sure thing. Any clue on the 304s?

Comment (rock3r) [reply functions/src/strings.ts]: ef6f8ff

**Pull request merged**